client/components/splash_codes/_edit.html

```diff
@@ -8,7 +8,7 @@
   <md-divider></md-divider>
 </md-toolbar>
 
-<md-content class="md-padding" layout="column" layout-gt-sm='row'>
+<md-content class="md-padding" layout="column" layout-gt-sm='column'>
```

Comment:
@ariandora Could you confirm this was supposed to be changed to column and double check it's ok on small screen.